typing/typecore.ml

```diff
@@ -3722,7 +3715,7 @@ and type_label_access env srecord lid =
   let labels = Env.lookup_all_labels ~loc:lid.loc lid.txt env in
   let label =
     wrap_disambiguate "This expression has" (mk_expected ty_exp)
-      (Label.disambiguate () lid env expected_type) labels in
+      (Label.disambiguate () lid env expected_type Result.ok) labels in
```

Comment:
@garrigue (looking at this over my shoulder) would like the `filter` parameter to be optional. (I dislike this style but he seems rather insistent.)

Reply:
I made the parameter optional.